interfaces/repositories.go

```diff
@@ -35,7 +36,7 @@ func NewDbUserRepo(dbHandlers map[string]DbHandler) *DbUserRepo {
 }
 
 func (repo *DbUserRepo) Store(user usecases.User) error {
-	err := repo.dbHandler.Execute(fmt.Sprintf(`INSERT INTO users (id, user_name, player_id, personal_info)
+	_, err := repo.dbHandler.Execute(fmt.Sprintf(`INSERT INTO users (id, user_name, player_id, personal_info)
```

Comment:
Do I need to name the field user_id instead? (id is primary key of table "users")